### PR TITLE
add total stakers

### PR DESCRIPTION
**Pull request by MrPai**
Get the total quantity of parachain stakers.

### CODE REVIEW

**Comment by MrPai** [comment]
As discussed, will only use `TotalStakers` records how many times the stake() function be invoked.

**Comment by MrPai** [comment]
please help review the changes, @kaichaosun @GopherJ

File: pallets/liquid-staking/src/lib.rs [hunk 1]
```diff
@@ -160,6 +160,11 @@ pub mod pallet {
     #[pallet::getter(fn total_voucher)]
     pub type TotalVoucher<T: Config> = StorageValue<_, Balance, ValueQuery>;
 
+    /// The total quantity of stakers.
```
Comment (kaichaosun) [inline pallets/liquid-staking/src/lib.rs:163]:
```suggestion
    /// The total person-times of staking operations.
```

File: pallets/liquid-staking/src/lib.rs [hunk 2]
```diff
@@ -160,6 +160,11 @@ pub mod pallet {
...
+    #[pallet::storage]
+    #[pallet::getter(fn total_stakers)]
+    pub type TotalStakers<T: Config> = StorageValue<_, u128, ValueQuery>;
```
Comment (kaichaosun) [inline pallets/liquid-staking/src/lib.rs:166]: Suggest this name of storage: StakingPersonTimes

File: pallets/liquid-staking/src/lib.rs [hunk 3]
```diff
@@ -265,6 +270,11 @@ pub mod pallet {
                 Ok(())
             })?;
 
+            let _ = TotalStakers::<T>::try_mutate(|b| -> DispatchResult {
```
Comment (kaichaosun) [inline pallets/liquid-staking/src/lib.rs:273]: Should use mutate.

File: pallets/liquid-staking/src/tests.rs [hunk 4]
```diff
@@ -702,3 +702,21 @@ fn process_pending_unstake_for_max_should_fail() {
         );
     })
 }
+
+#[test]
+fn total_stakers_calculate_should_work() {
```
Comment (kaichaosun) [inline pallets/liquid-staking/src/tests.rs:707]: No need for a new test case, just modify an existing one.

**Comment by MrPai** [comment]
@kaichaosun I make some changes as you suggest, pls review again if it's ok now, thanks.

**Review by kaichaosun — APPROVED** [review]
👍

**Pull request merged**